planner/core/exhaust_physical_plans.go

```diff
@@ -1483,16 +1483,37 @@ func (p *LogicalWindow) exhaustPhysicalPlans(prop *property.PhysicalProperty) []
 	var byItems []property.Item
 	byItems = append(byItems, p.PartitionBy...)
 	byItems = append(byItems, p.OrderBy...)
+
+	concurrency := p.ctx.GetSessionVars().WindowConcurrency
+	if concurrency > 1 && prop.IsEmpty() { //TODO: implement a cost-based strategy
```

Comment:
Could we make it simple?
If concurrency > 1 && prop.IsEmpty() && NDV > 1, we can just use `WindowParallel` instead of calculating cost.